lib/prompt-app-definition.js

```diff
@@ -0,0 +1,59 @@
+const inquirer = require('inquirer');
+const path = require('path');
+
+let appDefinitionSettings;
```

Comment:
Why do we keep it around in a stateful manner here?

Reply:
Fixed - was probably because of the previous /bin setup